Library/Homebrew/test/service_spec.rb

```diff
@@ -49,6 +49,34 @@ def stub_formula(&block)
     end
   end
 
+  describe "#nice" do
+    it "throws for nice too low" do
+      f = stub_formula do
+        service do
+          run opt_bin/"beanstalkd"
+          nice(-21)
```

Comment:
```suggestion
          nice -21
```

Or does RuboCop complain about this?

Reply:
yeah... syntax